### ISSUE TITLE
godot 3?

### ISSUE THREAD

**Issue by zenogears** [StraToN/godot-dialogtreetools#12]
Plans to upgrade that works with godot 3?

**Comment by StraToN** [comment]
Hello @zenogears, thanks for your interest in this project.
Yes, I plan to port this to Godot 3. I haven't started it yet but I will do soon.

**Comment by zenogears** [comment]
Thanks. Waiting for it :)
I was trying upgrade, but now my programming skills are at low level :)
And i can import addon, but it not worked as well :)

**Comment by StraToN** [comment]
Well this addon is a little bit complicated because it is all nested scenes instanced by code, so it is a bit hard to read. Problem is that graphnodes hold all the data, so the whole graph grows too heavy pretty quickly.
I'll use this opportunity to remake it and keep the graph lighter.

**Comment by yochrisbolton** [comment]
Great looking project! Any updates on a planned 3.x port? Cheers!

**Comment by StraToN** [comment]
@finchMFG Look at the 3.0 branch :)
https://github.com/StraToN/godot-dialogtreetools/tree/3.0

**Comment by zenogears** [comment]
Yes :) There is a branch, but plugin still do not working in 3.0 godot :)
So we are still waiting for updates. Thanks.

**Comment by yochrisbolton** [comment]
Missed it! Brilliant, cheers!

**Comment by StraToN** [comment]
@zenogears Are you using 3.0-stable or 3.0-master? This way I can focus my tests on the right version.
I'll try to work on it this week.

**Comment by zenogears** [comment]
Hi! I am using Godot_v3.0.2-stable_x11.64 right now :)

**Comment by StraToN** [comment]
Alright ! I'll focus on current stable. Thanks !
